src/components/RewardsCard/index.tsx

```diff
@@ -25,22 +26,24 @@ export type RewardsCardProps = {
 
 function RewardsCard({
   rewardImage,
+  rewardType,
   claimsLeft,
-  decimals,
+  decimals = 0,
   i18n = {
     claimsLabel: 'Claims left'
   },
   isLoading = false,
   rewardName,
   amountPerUser
 }: RewardsCardProps) {
-  let rewardText = undefined
+  let rewardText = rewardName
+
   if (amountPerUser) {
     let numToClaim = undefined
     if (amountPerUser && decimals !== undefined && decimals !== null) {
       numToClaim = getDecimalNumberFromAmount(
         amountPerUser,
-        decimals ?? 0
+        decimals
```

Comment:
```suggestion
        decimals ?? 0
```

Reply:
Yeah, it was like this but then it was breaking all the calcs and returning NaN. I will check how to fix it.